components/brave_new_tab_ui/api/initialData.ts

```diff
@@ -22,7 +22,7 @@ export type PreInitialRewardsData = {
 
 export type InitialRewardsData = {
   onlyAnonWallet: boolean
-  adsEstimatedEarnings: boolean
+  adsEstimatedEarnings: number
```

Comment:
do we need migration path?

Reply:
@NejcZdovc - this is just typing the payload response, it is correct in State interface

Reply:
@NejcZdovc we only use this interface to type responses, the type is correct in the state interface, this does not persist